About.html

```diff
@@ -0,0 +1,116 @@
+    <script src="home.js" defer></script>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <div id="portfolio" class="wrapper">
+    <header class="nav-container">
+      <div class="top-hader">
+        <nav>
+        <ul class="uls">
+          <li class="social-list glow-on-hover"><a href="#"><img src="images/facebook(1).png" alt="facebook logo"></a></li>
+          <li class="social-list glow-on-hover"><a href="#"><img src="images/twitter.png" alt="twitter logo"></a>
+           </li>
+          <li class="glow-on-hover">English</li>
+          <li class="glow-on-hover">MyPage</li>
+          <li class="glow-on-hover">Layout</li>
+        </ul>
+       </nav>
+       </div>
+       <div class="header2">
+        
+        <nav class="nav">
+           <div class="logoabut">
+            <a href="index.html"> <img class="logo-img1" src="images/logo.png" alt="logo cc Global summit 2015" width="100" height="50"></a>
+             </div>
+             <figure class="humburger">
+              <span  class="bar"></span>
+              <span class="bar"></span>
+              <span class="bar"></span>
+             </figure>
+             <ul class="nav-items" id="nav-items">
+              <li><a href="" class="closebtn">x</a></li> 
+              <li><a href="index.html" class="nav-item ">Home</a> </li>
+              <li><a href="About.html" class="nav-item nav-actives">About</a> </li>
+              <li><a href="#program" class="nav-item">Program</a> </li>
+             <li><a href="#locate" class="nav-item">Sponsor</a> </li>
+             <li><a href="#locate" class="nav-item CcChampaing">CC Champaing</a> </li>
+            </ul>
+        </nav>
+      </div>
+    </header>
+  <main class="cultural-days-wrap" id="main">
+    <section class="about new">
+      <h3 class="about-tite2">"hello! sharing world </h3>
+      <h1 class="about-titel">WORLDWIDE EVENT PROMOTER</h1>
+      <div class="invite-wrap invite-wrap-about">
+        <p class="invite-msg global">International Conference on Climate Change aims to bring together  scientists, researchers and research scholars to exchange and share their experiences and research results.</p>
+      </div>
+      <p class="text-about">Please contact us per Email for any further questions about CC Global Summit 2015!</p>
+      <p class="locate">Conference@newa123</p> 
+    </section>
+    <section class="logo" id="logo" >
+      <h2 class="secondary-tittle logo-tittle">WORLD CONFERENCE ALERTS 2022, 2023 Logo</h2>
+      <div class="bar-icon"></div>
+      <p class="invite-msg logo-msg">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Minima explicabo inventore omnis necessitatibus molestias labore rerum beatae consequuntur, et eius vitae recusandae, quis quidem. Minus fugit excepturi aut nemo mollitia.</p>
+      <img class="logo-img" src="images/logo.png" alt="logo " width="572" height="206">
+    </section>
```

Comment:
- [ ] Kindly follow the given design. This section is not aligned properly to the center, this is because you have put `margin-left` on the whole section, which has caused your content to not be properly aligned.
- [ ] Also follow the font-size and font-weight so as to match the given design
- [ ] The logo should be given some margin and padding so as to match the given design, as well as it should have a rectangular border around it as in the given design 

Your design
![image](https://user-images.githubusercontent.com/81932478/204780040-778b86d7-97c7-4527-b8e0-9199c2ccc32a.png)

Given design
![image](https://user-images.githubusercontent.com/81932478/204780146-727d5c5c-ae86-4076-8fee-aceb52691f6a.png)

- [ ] The purpose of this capstone is for you to pick a topic of you your choice and to create a website around that, so kindly add relevant information to your webiste as a whole and remove this placeholder text from `lorem` as this is a requirement